analyses/snv-callers/scripts/03-calculate_tmb.R

```diff
@@ -242,63 +225,152 @@ if (opt$tcga) {
 # Add in metadata
 strelka_mutect_maf_df <- strelka_mutect_maf_df %>%
   dplyr::inner_join(metadata %>%
```

Comment:
I would hope we don't have samples in the MAF files that are not in metadata, but in case that were to happen, an `inner_join` makes sure this continues. But yes, this *should* essentially be a `left_join` unless `metadata` is messed up.